src/game-item.ts

```diff
@@ -0,0 +1,36 @@
+import { AbiItemModel } from '@harmony-js/contract/dist/models/types'
+import { ContractOptions } from '@harmony-js/contract/dist/utils/options'
+import BN from 'bn.js'
+import { HRC1155 } from './hrc1155'
+import { ContractProviderType } from './interfaces'
+
+export class GameItems extends HRC1155 {
+  constructor(address: string, abi: AbiItemModel[], provider: ContractProviderType, options?: ContractOptions) {
+    super(address, abi, provider, options)
+  }
```

Comment:
This is not need it, because you are using the super with the same information.

Reply:
Okay, I'll fix it!